pkg/runtime/plan/show_create.go

```diff
@@ -0,0 +1,109 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package plan
+
+import (
+	"context"
+	"fmt"
+	"strings"
+)
+
+import (
+	"github.com/pkg/errors"
+)
+
+import (
+	"github.com/arana-db/arana/pkg/dataset"
+	"github.com/arana-db/arana/pkg/mysql/rows"
+	"github.com/arana-db/arana/pkg/proto"
+	"github.com/arana-db/arana/pkg/resultx"
+	"github.com/arana-db/arana/pkg/runtime/ast"
+)
+
+var (
+	_ proto.Plan = (*ShowCreatePlan)(nil)
+)
+
+type ShowCreatePlan struct {
```

Comment:
`ShowCreatePlan` is a common name, There are a lot of `show create` scene.
```
show create table xxxx;
show create index xxxx;
....
```

I am not sure handle those scene uniformly or not?

Reply:
ShowCreatePlan work for all show-create statement, for example "show create index xxx", not only for show-create-table. This Name is match for ast.ShowCreate. In other issue of "show create" (e.g. , issue#239 ), we can enhance ShowCreatePlan.